web-app/src/pages/Login.vue

```diff
@@ -69,6 +70,11 @@ export default class Login extends Vue {
           code: this.$route.query.code,
         },
       });
+      // private async mounted() {
+      //   if (this.$route.query.code) {
+      //     await api.internalApi.auth.getToken(String(this.$route.query.code));
```

Comment:
Do we really need this kind of code structure? `api.internalApi.auth.getToken`. We can access `getToken` directly, without accessing `api.internalApi.auth`. Also you are making two-side imports/exports in http.ts, auth.ts, index.ts and so on. I don't really see sense of it.